routes/books/helpers/sortBooksByRating.js

```diff
@@ -0,0 +1,24 @@
+const BookCategories = require('../../../models/db/bookCategories');
```

Comment:
You can use absolute imports if you want:
`require("models/db/bookCategories.js");`